src/parse.ts

```diff
@@ -189,10 +204,25 @@ const parseRelationship = (variableName: string, originalObject: string): Formul
     })
 }
 
+const parseInstance = (variableName: string): FormulaIdentifierInfo[] => {
+  if ((variableName.match(/\./g) || []).length > 2) {
+    return []
+  }
+  if (getObject(variableName).endsWith(RELATIONSHIP_SUFFIX)) {
```

Comment:
This is the ticket of us not supporting custom relationships right?

Reply:
Nope. It means that if we see something like `Account_r.Whatever` then `Whatever` can't be an instance, it has to be a field.